conda/environments/cuxfilter_dev_cuda10.0.yml

```diff
@@ -1,13 +1,13 @@
 # name: cudf_dev10.0
 channels:
-- rapidsai
+- rapidsai-nightly
 - nvidia
 - conda-forge
 - anaconda
 dependencies:
 - cupy>=6*
 - python>=3.6,<3.8
-- cudf>=0.13
+- cudf=0.13
```

Comment:
Please also update https://github.com/rapidsai/cuxfilter/blob/branch-0.13/ci/release/update-version.sh#L55 so during release, the `>` isn't added back.

Reply:
done